frontend/src/components/navigate.jsx

```diff
@@ -108,6 +110,12 @@ export default function Navigate() {
                   <Icon className="me-1" size={0.75} path={mdiAccountCircle} />
                   {user.preferred_username}
                 </NavDropdown.Item>
+                {user.groups && user.groups.includes(owners) && (
+                  <NavDropdown.Item as={Link} to="/governor" className="small d-flex align-items-center p-1">
+                    <Icon className="me-1" size={0.75} path={mdiShield} />
+                    Admin
+                  </NavDropdown.Item>
+                )}
```

Comment:
This link to the admin page would be shown only to the logged in users that belong to the requested group. To those who are not signed in or to those who are signed in but do not belong to the requested group, this link to the admin page would not be shown.